Makefile

```diff
@@ -65,26 +65,46 @@ devnet-up:
 	@bash ./ops-bedrock/devnet-up.sh
 .PHONY: devnet-up
 
+testnet-up:
+	@bash ./ops-bedrock/testnet-up.sh
+.PHONY: testnet-up
+
 devnet-up-deploy:
 	PYTHONPATH=./bedrock-devnet python3 ./bedrock-devnet/main.py --monorepo-dir=.
 .PHONY: devnet-up-deploy
 
 devnet-down:
-	@(cd ./ops-bedrock && GENESIS_TIMESTAMP=$(shell date +%s) docker-compose stop)
+	@(cd ./ops-bedrock && GENESIS_TIMESTAMP=$(shell date +%s) docker-compose -f docker-compose-devnet.yml stop)
 .PHONY: devnet-down
 
+testnet-down:
+	@(cd ./ops-bedrock && GENESIS_TIMESTAMP=$(shell date +%s) docker-compose -f docker-compose-testnet.yml stop)
+.PHONY: testnet-down
+
 devnet-clean:
 	rm -rf ./packages/contracts-bedrock/deployments/devnetL1
 	rm -rf ./.devnet
-	cd ./ops-bedrock && docker-compose down
+	cd ./ops-bedrock && docker-compose -f docker-compose-devnet.yml down
 	docker image ls 'ops-bedrock*' --format='{{.Repository}}' | xargs -r docker rmi
 	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
 .PHONY: devnet-clean
 
+testnet-clean:
+	rm -rf ./packages/contracts-bedrock/deployments/devnetL1
+	rm -rf ./.devnet
+	cd ./ops-bedrock && docker-compose -f docker-compose-testnet.yml down
+	docker image ls 'ops-bedrock*' --format='{{.Repository}}' | xargs -r docker rmi
+	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
```

Comment:
This cleans up all named volumes, however since we don't name our node store volume, it will persist. This is preferable for testnet as we want it to persist because syncing has bandwidth costs.

Reply:
it is best practice to keep it this way for this iteration? or name the volume and not remove it on `clean`?

Reply:
i think @tuxcanfly is saying that his is fine as is for testnets

Reply:
yep, was wondering if we want to improve now or later